srcs/main.c

```diff
@@ -27,13 +27,34 @@
  */
 
 
-int	main(void)
+int	main(int ac, char *av)
 {
+	int	fd;
 	/* 1. assign signals to appropriate handlers
 	 * 2. infinite while loop
 	 *   a. get user input
 	 *   b. parse it
 	 *   c. execute it
 	 */
+	if (ac > 2)
+	{
+		// error message ?
```

Comment:
(something like) error_and_exit(WRONG_ARGC);
I'll add the error message.

Reply:
Cool! I'll add that after pulling your branch